core/src/main/java/org/elasticsearch/action/fieldstats/FieldStats.java

```
@@ -272,6 +335,9 @@ public final void writeTo(StreamOutput out) throws IOException {
      * otherwise <code>false</code> is returned
      */
     public boolean match(IndexConstraint constraint) {
+        if (hasMinMax == false) {
+            return false;
+        }
```

should we throw an exception if min/max info is not available?